marlowe-apps/Finder.md

```diff
@@ -57,51 +53,35 @@ Available options:
                            The port number of the chain-seek server's job API.
                            Can be set as the environment variable
-                           MARLOWE_RT_CHAINSEEK_COMMAND_PORT (default: 3720)
-  --chain-seek-query-port PORT_NUMBER
-                           The port number of the chain-seek server's query API.
-                           Can be set as the environment variable
-                           MARLOWE_RT_CHAINSEEK_QUERY_PORT (default: 3716)
+                           MARLOWE_RT_CHAINSEEK_COMMAND_PORT (default: 13720)
   --chain-seek-sync-port PORT_NUMBER
                            The port number of the chain-seek server's
                            synchronization API. Can be set as the environment
                            variable MARLOWE_RT_CHAINSEEK_SYNC_PORT
-                           (default: 3715)
-  --history-host HOST_NAME The hostname of the Marlowe Runtime history server.
-                           Can be set as the environment variable
-                           MARLOWE_RT_HISTORY_HOST (default: "127.0.0.1")
-  --history-command-port PORT_NUMBER
-                           The port number of the history server's job API. Can
-                           be set as the environment variable
-                           MARLOWE_RT_HISTORY_COMMAND_PORT (default: 3717)
-  --history-query-port PORT_NUMBER
-                           The port number of the history server's query API.
-                           Can be set as the environment variable
-                           MARLOWE_RT_HISTORY_QUERY_PORT (default: 3718)
-  --history-sync-port PORT_NUMBER
-                           The port number of the history server's
+                           (default: 13715)
+  --marlowe-sync-host HOST_NAME
+                           The hostname of the Marlowe Runtime marlowe-sync
+                           server. Can be set as the environment variable
+                           MARLOWE_RT_SYNC_HOST (default: "127.0.0.1")
+  --marlowe-sync-sync-port PORT_NUMBER
+                           The port number of the marlowe-sync server's
                            synchronization API. Can be set as the environment
-                           variable MARLOWE_RT_HISTORY_SYNC_PORT (default: 3719)
-  --discovery-host HOST_NAME
-                           The hostname of the Marlowe Runtime discovery server.
-                           Can be set as the environment variable
-                           MARLOWE_RT_DISCOVERY_HOST (default: "127.0.0.1")
-  --discovery-query-port PORT_NUMBER
-                           The port number of the discovery server's query API.
-                           Can be set as the environment variable
-                           MARLOWE_RT_DISCOVERY_QUERY_PORT (default: 3721)
-  --discovery-sync-port PORT_NUMBER
-                           The port number of the discovery server's
+                           variable MARLOWE_RT_SYNC_SYNC_PORT (default: 13724)
```

Comment:
These environment variables are already defined, can you please update these to align?

https://github.com/input-output-hk/marlowe-cardano/blob/main/scripts/local-docker.env#L6

Reply:
Fixed in dd7cbd5b7646c2d03f5ee88274dab8653a21aba8.